src/views/pages/cart.twig

```diff
@@ -90,8 +90,11 @@
                                             <h1 class="text-gray-900 leading-6 text-lg">
                                                 <a href="{{ item.url }}" class="text-base">{{ item.product_name }}</a>
                                             </h1>
-                                            <span class="text-sm text-gray-500 line-through item-regular-price {{ item.offer ? '' : 'hidden' }}">{{ item.product_price|money }}</span>
-                                            <span class="item-price {{ item.offer?'text-red-800':'text-sm text-gray-500' }}">{{ item.price|money }}</span>
+
+                                            <span class="text-sm text-gray-500 line-through item-regular-price {{ item.offer ? '' : 'hidden' }}" >{{ item.product_price|money }}</span>
+                                            <span class="text-sm text-gray-500 line-through item-original-price {{ item.is_on_sale ? '' : 'hidden' }}" id="original_price">{{ item.original_price|money }}</span>
+                                            <span class="item-price {{ has_discount ? 'text-sm text-gray-500' : 'text-red-800' }}" id="has_discount">{{ item.price|money }}</span>
```

Comment:
- From where `has_discount` coming? it think it should be coming from `item.*`, it's the same `item.offer`?